Makefile

```diff
@@ -39,6 +39,7 @@ check:
 	tests/pylint/runpylint.py
 
 test:
+	@which nosetests || (echo "*** Please install python-nose ***"; exit 2)
```

Comment:
I'd rather see this suggesting users to install ``nosetest``, possibly with the package name in the parentheses. People could try using pykickstart on other distributions too.